python/tskit/trees.py

```diff
@@ -5352,11 +5352,13 @@ def write_vcf(
                 n_dip_indv = int(ts.num_samples / 2)
                 indv_names = [f"tsk_{str(i)}indv" for i in range(n_dip_indv)]
                 with open("output.vcf", "w") as vcf_file:
-                    ts.write_vcf(vcf_file, ploidy=2, individual_names=indv_names)
+                    ts.write_vcf(vcf_file, individual_names=indv_names)
 
             Adding a second ``_`` (eg: ``tsk_0_indv``) is not recommended as
             ``plink`` uses ``_`` as the default separator for separating family
-            id and individual id, and two ``_`` will throw an error.
+            id and individual id, and two ``_`` will throw an error. In this
+            case, ploidy does not need to be specified, as it is implied by the
```

Comment:
I think this last sentence is a bit confusing when it comes directly after the note about plink. I think we just need to say that ``ploidy`` cannot be provided when there is individual data in the tree sequence in the parameter documentation down further.

I think the issue here is that these examples were run using msprime 0.x and may be out of date now.